Automation/src/test/java/projectJavaIO/ReadCSVFileBasses.java

```diff
@@ -0,0 +1,25 @@
+package projectJavaIO;
+
+public class ReadCSVFileBasses {
+	private String make;
+	private String model;
+	
+	public String getMake() {
+		return make;
+	}
+	public void setMake(String make) {
+		this.make = make;
+	}
+	public String getModel() {
+		return model;
+	}
+	public void setModel(String model) {
+		this.model = model;
+	}
+	
+	@Override
+	public String toString() {

```

Comment:
If the intention of this method is to output the values of all fields then be aware that if more fields are added then you'll have to update this method. This sort of violates OpenClose principle.